code-challenges.test.js

```diff
@@ -38,32 +58,135 @@ const num3 = -7
+    // Use .charAt(0) && .toUpperCase() to capitalize entire string
+    // Use .split to convert string back to array once capitalized
+// Return output is an array containing all capitalized words.
+
+// const arrayofCapitalization = (array) => {
+//     return array.join(randomNouns1,randomNouns2) + array.charAt(0).toUpperCase +array.split (" ")
+    
+//     }
+    
+   
+// }
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   Jest encountered an unexpected token
+
+//   Details:
+
+//   SyntaxError:
+
+// Note to Instructor: I'm at a lost here with this problem, I don't even understand why I'm getting a syntax error (on line 123) or how to correctly use/order these commands. It feels like this entire problem is in Chinese. Literally spent hours on it even though there's not much code. I really thought I was understanding Jest but I haven't even got one of these tests to come back clear and I'm not sure why. This explains why I have commented out my #2 - code lines, so I can still run tests for the other function problems.
 
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe(pickupVowel, () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+      expect(pickupVowel("vowelTester1")).toEqual(1)
+      expect(pickupVowel("vowelTester2")).toEqual(0)
+      expect(pickupVowel("vowelTester3")).toEqual(2)
+    })
+ })
+
 const vowelTester1 = "learn"
 // Expected output: 1
 const vowelTester2 = "academy"
 // Expected output: 0
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: pickupVowel is not defined
 
 // b) Create the function that makes the test pass.
+
+// Pseudocode: 
+// Declare a function called pickupVowel.
+// Set parameters of string
+// Use conditional to evalute string.
+// Return index of first vowel location. 
+
+// const pickupVowel = (vowels) => {
+//    return vowels.search(a,e,i,o,u)
+// }
+
+// const pickupVowel = (vowels) => {
+//    return vowels.fitler(a,e,i,o,u)
+//    return 1 || 0 || 2
+// }
+
+const pickupVowel = (vowels) => {
+    for (int i =0; i<vowelTester.length, i++) {
+        if ()
+    }
+    }
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   ReferenceError: Cannot access 'pickupVowel' before initialization
+
+// Note to Instructor: I was a bit unsure on how to go about this one through the Jest nomenclature. Found this one a bit frustrating because I initially tried using something I found in an online search but felt it was missing something, so I tried .filter then remembered its not an array, so then I started to do a for loop for it but I was unsure of the syntax and how to fill it out with the Jest variables. 
```

Comment:
Your hard work may not have solved the problem, but it is obvious you put in the time and know the steps to take.  You used your resources.  I think getting more familiar with the jest syntax as well as making sure you're passing variables and not hard coded numbers is a good focus.  Really great effort overall, Steven!

Reply:
Understood, these suggestions are on my radar!